test/unit/snowflake_config_test.js

```diff
@@ -20,7 +19,7 @@ describe('Snowflake Configure Tests', function () {
       ocspFailOpen: GlobalConfig.getOcspFailOpen(),
       keepAlive: GlobalConfig.getKeepAlive(),
       jsonColumnVariantParser: GlobalConfig.jsonColumnVariantParser,
-      xmlColumnVariantParser: GlobalConfig.xmlColumnVariantParser
+      xmlColumnVariantParser: GlobalConfig.xmlColumnVariantParser,
```

Comment:
It shouldn't be change

Reply:
Could you please confirm whether we need to do the trailing comma or not? Dominik said we need to do it ,so that is why I put the comma at the end. I edited it first.